apps/fulltextsearch.sh

```diff
@@ -6,6 +6,7 @@
 # shellcheck disable=2034,2059
 true
 SCRIPT_NAME="Full Text Search"
+export SCRIPT_EXPLAINER="Full Text Search provides Elasticsearch for Nextcloud, which makes it possible to search for text inside files."
```

Comment:
Really, you don't need to export it to use it in a function

Reply:
Since it's used in another script, best practice is to export it. 

We source the lib, but the app script itself isn't sourced, which means exporting a variable inside the app script is needed to be able to run it in the lib. 

Please prove me wrong.

Reply:
I am not sure how to prove you wrong by words. But I have tested that it works without export and unset.

Reply:
exporting and unsetting fells like a big hack which is not necessary!